scripts/utils.ts

```diff
@@ -179,13 +183,30 @@ export const getGasPriceFromEnv = (): ethers.BigNumber => {
 	return gasPrice;
 };
 
+export type SubgraphBetOptions = {
+	unsettledOnly?: Boolean;
+	maxResults?: Number;
+	payoutAtLt?: Seconds;
+};
+
 export async function getSubgraphBetsSince(
-	createdAtGt: Seconds
+	createdAtGt: Seconds,
+	options: SubgraphBetOptions = {}
 ): Promise<BetDetails[]> {
 	const timeString = Math.floor(createdAtGt);
+
+	const { unsettledOnly = false, maxResults = 1000, payoutAtLt } = options;
+
+	const whereClauses = [
+		`createdAt_gt: "${timeString}"`,
+		unsettledOnly ? ", settled_not: true" : "",
+		payoutAtLt ? `payoutAt_lt: ${payoutAtLt}` : ""
+	];
+	const whereClause = whereClauses.filter(Boolean).join(",");
```

Comment:
why filter `Boolean` ? `whereClause.join(",")` should produce the same result

Reply:
ah nvm, i see the ternaries in there

Reply:
It cleans up the empty strings and avoids extra commas:

`['foo','',false, null, 'bar'].join(',')` // `'foo,,false,,bar'`
vs
`['foo','',false, null, 'bar'].filter(Boolean).join(',')` // `'foo,bar'`

Reply:
:TIL: Looking at that output, it looks like we could use nulls in the ternaries instead. I wonder if I prefer that